src/main/java/com/agency04/sbss/pizza/service/impl/PizzaDeliveryServiceImpl.java

```diff
@@ -83,35 +75,36 @@ public PizzeriaMenu getPizzeriaMenu() {
 
 	@Override
 	public List<PizzaOrder> getCurrentOrders() {
-		return this.currentOrders;
-	}
+		Delivery delivery = deliveryRepository.getTopByOrderBySubmissionDateDesc();
 
-	public void setCurrentOrders(List<PizzaOrder> currentOrders) {
-		this.currentOrders = currentOrders;
+		if (delivery != null) {
+			return delivery.getPizzaOrders();
+		} else
+			throw new NullPointerException("No deliveries ordered yet.");
```

Comment:
Zašto se baca NPE? To bi imalo smisla kada bi se pojedinačno tražio neki specifičan order (ili recimo Customer po usernameu) pa nije pronađen (iako bi tada trebalo baciti neki custom exception, nikako ne NPE). U slučaju kada se dohvaća lista nečega i ako je ona prazna vrati se status 200 s praznom listom.